vmm/src/api/http.rs

```diff
@@ -86,8 +89,21 @@ pub enum HttpError {
     VmAddVsock(ApiError),
 }
 
+impl From<serde_json::Error> for HttpError {
+    fn from(e: serde_json::Error) -> Self {
+        HttpError::SerdeJsonDeserialize(e)
+    }
+}
+
 const HTTP_ROOT: &str = "/api/v1";
 
+pub fn error_response(error: HttpError, status: StatusCode) -> Response {
```

Comment:
This function should have been moved as part of the previous commit.

Reply:
I did intent for it to be in the commit because it was part of the `put_handler` rather than related to moving the enum. But I can see that it would also have made sense in the previous one.

Reply:
ok that's fine then :)